cuqi/experimental/mcmc/_pcn.py

```diff
@@ -64,17 +64,19 @@ def likelihood(self):
 
     @SamplerNew.target.setter 
     def target(self, value):
-        if isinstance(value, cuqi.distribution.Posterior):
-            self._target = value
-            self._loglikelihood = lambda x : self.likelihood.logd(x)
-        elif isinstance(value,tuple) and len(value)==2 and \
-             (isinstance(value[0], cuqi.likelihood.Likelihood) or isinstance(value[0], cuqi.likelihood.UserDefinedLikelihood))  and \
-             isinstance(value[1], cuqi.distribution.Distribution):
-            self._target = value
-            self._loglikelihood = lambda x : self.likelihood.logd(x)
+        if value is not None:
+            if isinstance(value, cuqi.distribution.Posterior):
+                self._target = value
+                self._loglikelihood = lambda x : self.likelihood.logd(x)
+            elif isinstance(value,tuple) and len(value)==2 and \
+                (isinstance(value[0], cuqi.likelihood.Likelihood) or isinstance(value[0], cuqi.likelihood.UserDefinedLikelihood))  and \
+                isinstance(value[1], cuqi.distribution.Distribution):
+                self._target = value
+                self._loglikelihood = lambda x : self.likelihood.logd(x)
+            else:
+                raise ValueError(f"To initialize an object of type {self.__class__}, 'target' need to be of type 'cuqi.distribution.Posterior'.")
         else:
-            raise ValueError(f"To initialize an object of type {self.__class__}, 'target' need to be of type 'cuqi.distribution.Posterior'.")
-        
+            self._target = value
```

Comment:
target here does not seem call validate_target. Should we instead but the logic of this method in validate_target instead of overwriting the target setter?

Reply:
Ah yes. I guess I was a bit lazy when going over this existing code. I now simplified to 
```python
    def _loglikelihood(self, x):
        return self.likelihood.logd(x)
```
removing the option to define target as tuple ( similar to other simplifications we have done)

Reply:
This also simplified the validate_target method.

Reply:
nice simplification!